### PR TITLE
Change list formatting for "Challenge problems" section in "Scipy.integrate" tutorial 

### LINKED ISSUE #320: "Using scipy.integrate" tutorial has some weirdness in the display
"[Using scipy.integrate](http://www.astropy.org/astropy-tutorials/rst-tutorials/units-and-integration.html)" has a few problems that show up in the website display, but that look fine in the actual notebook:
- The cell that contains imports in the [summary section](http://www.astropy.org/astropy-tutorials/rst-tutorials/units-and-integration.html?highlight=filtertutorials#summary) is not displaying at all
- The links in the [How to simulate actual observations](http://www.astropy.org/astropy-tutorials/rst-tutorials/units-and-integration.html#how-to-simulate-actual-observations) section are showing their raw markdown instead of looking like proper links (Also in the [Integrating Using Gaussian Quadrature](http://www.astropy.org/astropy-tutorials/rst-tutorials/units-and-integration.html#integrating-using-gaussian-quadrature) section)
- In the [Challenge Problems](http://www.astropy.org/astropy-tutorials/rst-tutorials/units-and-integration.html#challenge-problems) section some of the latex is rendered but some shows up as raw latex
- The "Download Tutorial Notebook" and "Interactive Tutorial" buttons at the top of the notebook work, but the ones at the bottom do not.

### PR DESCRIPTION
**Pull request by kakirastern**
Following up on previous work. To change the formatting of the list of questions in the "Challenge problems" section (i.e. the last section at the bottom) in the "Scipy.integrate" tutorial to tidy things up. 

Fixes #320.

### CODE REVIEW

**Comment by kakirastern** [comment]
After this PR will have been merged to the code, the formatting for the scipy.integrate tutorial as seen in web browser will be fixed, at least for the time being.

File: tutorials/notebooks/units-and-integration/units-and-integration.ipynb [hunk 1]
```diff
@@ -326,14 +324,16 @@
     "\n",
     "* Right now, we aren't worried about the bounds of the power law, but the IMF should drop off to zero probability at masses below .01 solar masses and above 100 solar masses. Modify `PowerLawPDF` in a way that allows both `float` and `numpy.ndarray` inputs.\n",
     "* Modify the `PowerLawPDF` class to explicitly use `astropy`'s `units` constructs.\n",
-    "* Derive a relationship between recent star-formation rate and $H\\alpha$ luminosity. In other words, for the function\n",
+    "* Derive a relationship between recent star-formation rate and $H\\alpha$ luminosity.\n", 
+    "* In other words, find a value of $C$ for the function\n",
```
Comment (adrn) [inline tutorials/notebooks/units-and-integration/units-and-integration.ipynb]: Leave this as one line

File: tutorials/conf.py [hunk 2]
```diff
@@ -96,7 +96,7 @@
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
-html_favicon = 'astropy_favicon.ico'
+html_favicon = "_static/astropy_favicon.ico"
```
Comment (adrn) [inline tutorials/conf.py]: Can you rebase to remove these other commits?

File: tutorials/notebooks/units-and-integration/units-and-integration.ipynb [hunk 3]
```diff
@@ -301,7 +301,7 @@
    "source": [
     "### Extras\n",
     "\n",
-    "* Now compare the total luminosity from all O stars to total luminosity from all M stars. This requires a mass-luminosity relation, like this one:\n",
+    "* Now compare the total luminosity from all O stars to total luminosity from all M stars. This requires a mass-luminosity relation, like this one which you will use as $\\rho(m)$:\n",
```
Comment (adrn) [inline tutorials/notebooks/units-and-integration/units-and-integration.ipynb]: Put a comma before "which"

File: tutorials/notebooks/units-and-integration/units-and-integration.ipynb (hunk 1 above)
Comment (kakirastern) [reply tutorials/notebooks/units-and-integration/units-and-integration.ipynb]: Yup, done.

File: tutorials/conf.py (hunk 2 above)
Comment (kakirastern) [reply tutorials/conf.py]: Yup, done.

File: tutorials/notebooks/units-and-integration/units-and-integration.ipynb (hunk 3 above)
Comment (kakirastern) [reply tutorials/notebooks/units-and-integration/units-and-integration.ipynb]: Yup, done.

**Comment by kakirastern** [comment]
I will rebase soon.

**Pull request merged**